traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/cache/Cache.java

```diff
@@ -101,9 +103,10 @@ public String getId() {
 			if(ttls == null) {
 				ttl = -1;
 			} else if(ir.isInet6()) {
-				ttl = ttls.optLong("AAAA");
+				ttl = JsonUtils.getLong(ttls, "AAAA", 0);
```

Comment:
Should probably have a default here..

Reply:
The third argument of the JsonUtils.get methods is the default in case the key isn't in the JsonNode object